webview-ui/src/i18n/locales/it/welcome.json

```diff
@@ -1,5 +1,5 @@
 {
-	"greeting": "Ciao, sono Roo!",
+	"greeting": "Welcome to Roo",
```

Comment:
```suggestion
	"greeting": "Welcome to Roo Code",
```

Reply:
Soft pushback: I suggest sticking with just 'Roo' here. Might be a branding standards item for the team to discuss, but once users are this far in the onboarding flow, approachability/friendliness are a huge asset (imo).